lib_eio_windows/test/test_fs.ml

```diff
@@ -130,69 +125,68 @@ let test_mkdir env () =
   Unix.rmdir "subdir\\nested";
   Unix.rmdir "subdir"
 
-let test_mkdirs env () =

```

Comment:
Looks like we might have forgotten to add this test to the test-suite. Instead of removing it can we add it at the bottom?

Reply:
Alright. I'll fix at the bottom and see how it goes